packages/sdk-codegen/package.json

```diff
@@ -33,9 +33,10 @@
     "watch": "yarn lerna exec --scope @looker/sdk-codegen --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments --watch'"
   },
   "dependencies": {
-    "@looker/sdk-codegen-utils": "^21.0.9",
-    "@looker/sdk-node": "^21.0.9",
-    "@looker/sdk-rtl": "^21.0.9",
+    "@looker/sdk": "^21.0.10",
```

Comment:
see my other comment about hopefully dropping this dependency

Reply:
Yup. Let's put up a new PR for that. Drop `@looker/sdk` and remove it from sdk-codegen imports and see where we're at!